hack/deploy-seed.sh

```diff
@@ -8,6 +8,10 @@ echo ""
 echo "Installing Grafana"
 helm --namespace mla upgrade --atomic --create-namespace --install grafana charts/grafana --values config/grafana/values.yaml
 
+echo ""
+echo "Installing Grafana Dashboards"
+kubectl apply -f dashboards/
+
```

Comment:
Can you please also add this to the `hack/ci/deploy-dev.sh` so dashboards will be automatically rollout in dev cluster